### DISCUSSION TITLE
IEEE address

### DISCUSSION THREAD

**Discussion by chris-1243** [darkxst/silabs-firmware-builder#120] [Ideas]
Would it be possible to add the possibility to write a secondary ieee address in order to change easily a coordinator without the pain to re-pair everything ? Backup are now supported in `ember` while using Zigbee2mqtt and in `ezsp` as well. You need v12 firmware in order to get a backup

**Comment by darkxst** [comment]
With recent firmwares the IEEE address is now stored as a token in NV memory, rather than being burned into OTP memory. I believe in ZHA (via bellows) this is handled automatically on restoring backups. I not sure of the status in `ember`, but it should be handled there (and not via the firmware flasher).

**Comment by chris-1243** [comment]
I am going to ask on the Zigbee2mqtt Github.

I hope the TI Chip based on `zstack` would use the same standard. Backups would be easier to manage from different chips.

Thanks for your time and work

**Reply by tl-sl** [reply]
No TI chips are different again, there its stored in the Customer Config area with is essentially still part of the firmware.
